deploy/FlapperInit.sol

```diff
@@ -85,16 +103,13 @@ library FlapperInit {
         flapper.file("hop",  cfg.hop);
         flapper.file("want", cfg.want);
         flapper.file("pip",  cfg.pip);
-        flapper.rely(address(dss.vow));
         flapper.rely(address(mom));
 
-        dss.vow.file("flapper", address(flapper));
         dss.vow.file("hump",    cfg.hump);
         dss.vow.file("bump",    cfg.bump);
 
         mom.setAuthority(dss.chainlog.getAddress("MCD_ADM"));
 
-        dss.chainlog.setAddress("MCD_FLAP",    address(flapper));
```

Comment:
I think the low level flapper (burner) should have a representation in the chainlog, as it is authed by the PP (and its `hop` parameter basically sets the cadence of the whole thing).

Reply:
What do you guys think of keeping `MCD_FLAP` for whatever is set as `flapper` in the `vow` and use more specific names for all potential flappers / flapper modules (e.g. `MCD_FLAP_SPLIT` for the splitter, `MCD_FLAP_LP` for the current flapper and `MCD_FLAP_BURN` for the future burner)?

Reply:
will defer to @sunbreak1211 for this one :)

Reply:
I think the issue is that we are trying to support both a mode without a splitter and a mode with one. So not sure how removing `MCD-FLAP` fits into that. Maybe if we do that as part of `initSplitter`.

Regardless, note that in the current code we don't really need the setting of `MCD_FLAP` explicitly in `initFlapperUniV2()` and overriding in `initSplitter()`, as even if we are just using a flapper without a splitter we can pass `MCD_FLAP` as `cfg.chainlogKey`.